RUFAS/routines/animal/animal_module_reporter.py

```diff
@@ -29,6 +29,7 @@ def data_padder(
         thing_to_add: Any,
         simulation_day: int,
         info_map: Dict[str, Any],
+        units: Dict[str, str | MeasurementUnits] | MeasurementUnits | str,
```

Comment:
Because of the way units are nested within info_maps and not wanting to extract and change them from their enums to their enum values later for reporting, I ended up using the enum values (which are the actual strings for the units). When handling units is finished, we may need to go back and change these to the Enums but in the meantime, these type hints should be for strings.
```suggestion
        units: Dict[str, str] | str,
```